install_scripts/fedora_camel_toolbox.sh

```diff
@@ -35,9 +35,11 @@ if [ ! -f "/opt/karaf/etc/org.fcrepo.camel.audit.cfg" ]; then
 fi
 
 # Fixity service
+sleep 10
 if [ ! -f "/opt/karaf/etc/org.fcrepo.camel.fixity.cfg" ]; then
    /opt/karaf/bin/client -u karaf -h localhost -a 8101 "feature:install fcrepo-fixity"
 fi
+sleep 10
```

Comment:
Maybe this should be inside the ```if then ... fi``` so we only wait if you had to perform the feature install.